Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -134,6 +134,25 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
                     bool mergeIssue = false;
                     bool notifyReplaceAPI = false;
                     AddAPIModelWizard.LearnedAPIModelsList.Clear();
+                    List<DeltaAPIModel> repeatingMatchingModels = AddAPIModelWizard.DeltaModelsList.Where(m => m.SelectedOperationEnum == DeltaAPIModel.eHandlingOperations.ReplaceExisting ||
+                                                                m.SelectedOperationEnum == DeltaAPIModel.eHandlingOperations.MergeChanges)
```

Comment:
no, i don't want to have the duplicate API match validation only when user shift page because by that point he might already worked on configuring unvalid merges and only at end we telling him it is not valid.

you need to implment as following:
1- when we do the auto match as part of "Compare" process we should not set match for API which is already already was set as match to other API
2- if user select/edit the match and selecting API which is already matching to other API so cancel the change and show pop up saying the select API is already configured as match to another